lib/cdo/activity_constants.rb

```diff
@@ -14,6 +14,14 @@ module ActivityConstants
   MANUAL_PASS_RESULT = 90
   BEST_PASS_RESULT = 100
 
+  PASSING_VALUES = [
+    MINIMUM_PASS_RESULT,
+    MAXIMUM_NONOPTIMAL_RESULT,
+    FREE_PLAY_RESULT,
+    MANUAL_PASS_RESULT,
+    BEST_PASS_RESULT
+  ]
```

Comment:
This array scares me, these values should be such that `result >= MINIMUM_PASS_RESULT` guarantees that `result` is passing. If that isn't the case, we should fix that.